app/frontend/components/pages/exchange/exchange.tsx

```diff
@@ -0,0 +1,25 @@
+          >
+            Can't load widget
+          </iframe>
+        </div>
+      </div>
+    </div>
+  </div>
+)
+
```

Comment:
let's use this thread so that the topmost level is not spammed - Did you already test this as Michal asked?

Reply:
There is apparently minimum ADA required for a transaction. Do we have an account with more then 35 ADA?